composer.json

```diff
@@ -26,7 +26,8 @@
         "sebastian/version": "^2.0",
         "symfony/console": "^2.8 || ^3.0 || ^4.0",
         "symfony/yaml": "^2.8 || ^3.1 || ^4.0",
-        "mehr-als-nix/parallel": "^v1.0"
+        "mehr-als-nix/parallel": "^v1.0",
+        "jawira/plantuml-encoding": "^1.0"
```

Comment:
The dependency should be moved to the require-dev section for testing.
Could you also add a suggest entry for your lib?

Reply:
it should be fixed with my last commit